inst/examples/showcase/app.R

```diff
@@ -153,7 +158,38 @@ makeRouter <- function(items) {
       server = function() example$server(item$id)
     )
   })
-  do.call(shiny.router::make_router, routes)
+  routes <- append(
+    list(route(
+      path = "/",
+      ui = div(
+        class = "home-page",
+        H2(
+          class = "home-page-title",
+          "This is a ",
+          tags$a(
+            "Blueprint",
+            class = "weight-300",
+            href = "https://blueprintjs.com/",
+            target = "_blank"
+          ),
+          " app built in Shiny"
+        ),
+        span(class = "font-mono", "shiny.react + Blueprint = shiny.blueprint"),
+        Card(
+          class = "home-page-section",
+          span("Welcome to ", tags$strong(class = "font-mono", "shiny.blueprint"), " demo!"),
+          span(
+            tags$strong(class = "font-mono", "shiny.blueprint"), " is a package that allows ",
+            "you to build Shiny apps using Bleprint - a React-based UI toolkit for the web."
+          ),
+          span("Use the menu on the left to explore live demos of all available components.")
+        )
```

Comment:
[optional] add icons for the 2 libraries

```suggestion
        ),
        Card(
          class = "home-page-section",
          span(
            tags$strong(class = "font-mono", "Shiny.react"),
            " R package enables using React in Shiny apps.",
            "It contains R and JS code which is independent from the React library that is being wrapped."
          ),
          a(
            href = "https://appsilon.github.io/shiny.react/",
            target = "_blank",
            img(
              class = "logo",
              src = "https://github.com/Appsilon/shiny.react/raw/master/man/figures/shiny-react.png"
            )
          )
        ),
        Card(
          class = "home-page-section",
          span(
            tags$strong(class = "font-mono", "Blueprint"),
            " is a React-based UI toolkit for the web.",
            "It is optimized for building complex, data-dense web interfaces for desktop applications",
            "which run in modern browsers and IE11. This is not a mobile-first UI toolkit."
          ),
          a(
            href = "https://blueprintjs.com/",
            target = "_blank",
            img(
              class = "logo",
              src = "https://cloud.githubusercontent.com/assets/464822/20228152/d3f36dc2-a804-11e6-80ff-51ada2d13ea7.png"
            )
          )
        )
```

Reply:
Cards with description and logos added